### DISCUSSION TITLE
No TSO on Mac M1 Pro?

### DISCUSSION THREAD

**Discussion by christophenouvel** [utmapp/UTM#6731] [UTM Help (macOS)]
Hi,

Running UTM on a Mac M1 Pro since 09/2022, emulating a production x86 Windows 10 VM with encrypted FS + VPN all day long with no problem. Mainly remote desktop. One day per week, every week.

4.5.4 is rock solid.

But slow...

So as soon as 4.6.0 beta was released, I upgraded MacOS to Sequoia(15.0.1), in order to try TSO (and, hopefully : speedup my emulated VM).

But I can't see any "TSO" checkbox / option in QEMU?

M1 Pro is not enough?

*****

UTM : 4.6.0 beta
macOS version : Sequoia 15.0.1
Mac Chip : M1 Pro, 16GB RAM.

  Regards,

    Christophe.

**Comment by osy** [comment]
TSO only works on virtualization (ARM64 VM) and requires support from the OS (install Rosetta for Linux)
